.github/workflows/testing.yml

```diff
@@ -24,12 +24,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      max-parallel: 1
+      max-parallel: 2
       matrix:
-        jest_env: ['node', 'edge']
         pinecone_env:
           - prod
           # - staging
+        config:
+          [
+            { runner: 'npm', jest_env: 'node' },
+            { runner: 'npm', jest_env: 'edge' },
+            { runner: 'bun', jest_env: 'node', bun_version: '1.0.4' },
+            { runner: 'bun', jest_env: 'node', bun_version: '1.0.5' },
+            { runner: 'bun', jest_env: 'node', bun_version: '1.0.6' },
+            { runner: 'bun', jest_env: 'node', bun_version: '1.0.7' },
```

Comment:
The bun team seems to ship all the time. Since I started this work a couple months ago they shipped like 15 more versions are up to `1.0.22`. It's probably not reasonable to test every single one of those separately, but you could choose some representative numbers throughout the range.

https://github.com/oven-sh/bun/releases

Reply:
Good call, also whoa that's a very rapid release cadence. I'll spread the range out a bit more, thanks.

Reply:
Added one more step in the matrix, we're now hitting bun `1.0.0`, `1.0.5`, `1.0.10`, `1.0.15`, and `1.0.20`.